src/main/java/com/kodilla/ecommercee/domain/user/User.java

```diff
@@ -0,0 +1,49 @@
+package com.kodilla.ecommercee.domain.user;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.persistence.*;
+import java.util.Random;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+public class User {
+    @Id
+    @Column(name = "ID", unique = true)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "USERNAME")
+    private String userName;
+
+    @Column(name = "STATUS")
+    private boolean status;
+
+    @Column(name = "USER_KEY")
+    private int userKey;
+
+    public User(String userName){
+        this.userName = userName;
+        status = true;
+        userKey = keyGenerator();
+    }
+
+    public int keyGenerator(){
+        int keyValue = 0;
+        Random generator = new Random();
+        int[] keyNumbers = new int[5];
+        for (int i = 0 ; i < 5 ; i++ ){
```

Comment:
spróbuj pętle for zamienić streamami (np. IntStream), tak aby kod był bardziej funkcyjny